### PR TITLE
Add Information about Access Logs

### PR DESCRIPTION
**Pull request by MarkvanMents**
[TW-2003]/[LM-1189]

### CODE REVIEW

File: content/en/docs/apidocs-mxsdk/apidocs/deploy-api/_index.md [hunk 1]
```diff
@@ -1072,7 +1061,7 @@ Mendix-ApiKey:  26587896-1cef-4483-accf-ad304e2673d6
 []
 ```
 
-### 3.19 Download Archived Logs for a Specific Date (Mendix Cloud v4 Only)
+### 3.19 Download Archived Logs for a Specific Date
```
Comment (hsaeleven) [inline content/en/docs/apidocs-mxsdk/apidocs/deploy-api/_index.md]: ''Archived Logs'' are called '' App Logs'' per this change. Could you please change that in this document. See also  the other change where Archived was changed to App.

Comment (MarkvanMents) [reply content/en/docs/apidocs-mxsdk/apidocs/deploy-api/_index.md]: Updated in this commit: https://github.com/mendix/docs/pull/5569/commits/8315b1fb0c8fd7b098e0ef95b1349de659b7a6f1

**Review by hsaeleven — APPROVED** [review]
Looks good, Cp changes go live on Thursday.

**Comment by udh-mendix** [comment]
LGTM

**Review by udh-mendix — APPROVED** [review]
LGTM

**Pull request merged**